tvtk/tests/test_tvtk.py

```diff
@@ -647,34 +842,179 @@ def tearDown(self):
+            get_max_method = 'Get' + vtk_attr_name + 'MaxValue'
+            try:
+                return (getattr(vtk_klass(), get_min_method)(),
+                        getattr(vtk_klass(), get_max_method)())
+            except AttributeError:
+                return None, None
+
+        for name in self.names:
+            vtk_klass = getattr(vtk, name)
+            tvtk_klass_name = get_tvtk_name(name)
+
+            try:
+                obj = getattr(tvtk, tvtk_klass_name)()
+            except Exception:
+                # testing for instantiation is above
+                pass
+
+            for trait_name in obj.editable_traits():
+                vtk_attr_name = to_camel_case(trait_name)
+                min_value, max_value = get_min_max_value(vtk_klass,
+                                                         vtk_attr_name)
+
+                # Explicitly checking for None instead of bool(value)
+                # since min_value/max_value could be int(0)
+                if max_value is not None and min_value is not None:
+                    # If max and min values are defined, setting the trait
+                    # to outside this range should fail
+                    with self.assertRaises(TraitError):
+                        setattr(obj, trait_name, (min_value-1, max_value))
+                    with self.assertRaises(TraitError):
+                        setattr(obj, trait_name, (min_value, max_value+1))
+
+    def test_no_trait_has_ptr_address_as_value(self):
+        '''Test if none of the TVTK classes' traits has a value of "*_p_void"
+        '''
+        errors_getting_trait = []
+        errors_trait_is_ptr = []
+        for name in self.names:
+            tvtk_klass_name = get_tvtk_name(name)
+
+            try:
+                obj = getattr(tvtk, tvtk_klass_name)()
+            except Exception:
+                # testing for instantiation is above
+                pass
+
+            for trait_name in obj.editable_traits():
+                try:
+                    trait = getattr(obj, trait_name)
+                except (TypeError, TraitError):
+                    errors_getting_trait.append(
+                        (tvtk_klass_name, trait_name))
+                else:
+                    if isinstance(trait, str) and trait.endswith('_p_void'):
+                        errors_trait_is_ptr.append(
+                            (tvtk_klass_name, trait_name, trait))
+
+        if errors_trait_is_ptr:
+            message = ('These traits cannot be obtained:\n'
+                       '\n'.join(('{0}.{1}'.format(*error)
+                                  for error in errors_getting_trait)))
+        if errors_trait_is_ptr:
+            message = ('These traits are invalid:\n'
+                       '\n'.join(('{0}.{1} = {2}'.format(*error)
+                                  for error in errors_trait_is_ptr)))
+
+    def test_no_trait_has_ptr_address_as_value(self):
```

Comment:
The first `test_no_trait_has_ptr_address_as_value` is a ghost copypasta that was not deleted, deleting it now.